### ISSUE TITLE
Remove compiled app?

### ISSUE THREAD

**Issue by rovo89** [ClemensElflein/open_mower_ros#94]
Having the compiled app as part of the repository makes it harder to manage PRs which contain changes to the UI. And the app is definitely no part of the ROS packages.

So I'm wondering whether you'd agree to remove the web directory completely, and compile it while building the image (i.e. make the app build another stage of the Dockerfile).

It might make even more sense to compile the app after pushing updates to the app repository, but I couldn't think of a good way to make those artifacts available.

If you want to explicitly refer to a certain app version, it could be included as a submodule.

**Comment by ClemensElflein** [comment]
Yes I agree, the reason for this was to sync the app version as you said and also I didn't release the sources at first, but now since they are open, it makes sense to move it.

Note that I'm thinking about reworking the app completely. There will be a container which has a web UI and allows the user to select the `open_mower_ros` container version and change the settings file via web interface. This container will also launch the actual ros container (I'll mount the podman socket into it).

In order to not have two apps, I'm thinking about deprecating the current one and adding the functionality to the new one. I'm not sure how I'd keep those in sync, but with a sane openapi.yaml I think it's not an issue.
